CRM/Utils/System/Joomla.php

```diff
@@ -38,7 +38,7 @@ public function __construct() {
    * @return bool
    */
   public function isLoaded(): bool {
-    return class_exists('JFactory');
+    return class_exists('JFactory') || class_exists('Joomla\CMS\Factory');
```

Comment:
Properly, I think this should have a preceding backslash, i.e. `class_exists('\Joomla\CMS\Factory')`, just in case we are somehow not in the root namespace.